tldw_Server_API/app/core/Ingestion_Media_Processing/XML_Ingestion_Lib.py

```
@@ -159,23 +155,26 @@ def import_xml_handler(import_file, title, author, keywords, system_prompt,
+                url=display_name,  # Using filename as URL
+                info_dict=info_dict,
+                segments=segments,
+                summary=summary,
+                keywords=keyword_list,
+                custom_prompt_input=custom_prompt,
+                whisper_model="XML Import",
+                media_type="xml_document",
+                overwrite=False
+            )
 
         return f"XML file '{display_name}' import complete. Database result: {result}"
```

Ruff (0.15.6)

[warning] Consider moving this statement to an `else` block

(TRY300)